Source/WebInspectorUI/UserInterface/Views/ComputedStyleSection.js

```diff
@@ -221,6 +244,14 @@ WI.ComputedStyleSection = class ComputedStyleSection extends WI.View
             expandableView.element.classList.add("computed-property-item");
             this.element.append(expandableView.element);
         }
+
+        if (this._filterText || this._filterCleared)
+            this.dispatchEventToListeners(WI.ComputedStyleSection.Event.FilterApplied, {matches: !this.isEmpty});
+
+        if (this._filterCleared)
+            this._filterCleared = false;
```

Comment:
why not just this?
```
this._filterCleared = false;
```

Reply:
For clarity, really. But the standalone assignment works the same. Fixed.